src/vstart.sh

```diff
@@ -748,21 +748,23 @@ EOF
 		mkdir -p $CEPH_DEV_DIR/osd$osd
 	    fi
 
-            local uuid=`uuidgen`
-            echo "add osd$osd $uuid"
-	    OSD_SECRET=$($CEPH_BIN/ceph-authtool --gen-print-key)
-	    echo "{\"cephx_secret\": \"$OSD_SECRET\"}" > $CEPH_DEV_DIR/osd$osd/new.json
-            ceph_adm osd new $uuid -i $CEPH_DEV_DIR/osd$osd/new.json
-	    rm $CEPH_DEV_DIR/osd$osd/new.json
-            $SUDO $CEPH_BIN/ceph-osd -i $osd $ARGS --mkfs --key $OSD_SECRET --osd-uuid $uuid
-
-            local key_fn=$CEPH_DEV_DIR/osd$osd/keyring
-	    cat > $key_fn<<EOF
-[osd.$osd]
-	key = $OSD_SECRET
+        local key_fn=$CEPH_DEV_DIR/osd$osd/keyring
```

Comment:
@iotcg could you be more specific in the commit message? "refine" is kind of vague in this context. how about 

> add newly created osd's auth before mkfs to avoid following warning message:

?